package.json

```diff
@@ -41,11 +42,18 @@
   "devDependencies": {
     "@stencila/dev-config": "1.4.129",
     "@types/jest": "26.0.20",
+    "benchmark": "2.1.4",
+    "bole": "4.0.0",
+    "bunyan": "1.8.15",
     "jest": "26.6.3",
+    "log4js": "6.3.0",
+    "loglevel": "1.7.1",
     "microbundle": "0.13.0",
+    "pino": "6.10.0",
     "ts-jest": "26.4.4",
     "typedoc": "0.20.14",
-    "typescript": "4.1.3"
+    "typescript": "4.1.3",
+    "winston": "3.3.3"
```

Comment:
I would prefer not to commit all the various logging libraries to the project. I think just having the baseline numbers as you've documented in the PR comment is enough to compare against.
There are several downsides of having them in here, but the primary desire for me would be to reduce the potential for conflicting package dependencies (especially painful with type definitions), as well as overhead in keeping things up to date.

Reply:
Agreed (and comment expected :)). I'll remove them and just add a comment to the `benchmark.js` about installing them.